CHANGELOG.md

```diff
@@ -10,33 +10,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Fixed an issue with TopOverlay position for RTL [#8764](https://github.com/handsontable/handsontable/issues/8764)
-- Fixed an issue with numeric cell formatting for RTL [#8791](https://github.com/handsontable/handsontable/issues/8791)
-- Fixed an issue with htAutocompleteArrow position for RTL [#8795](https://github.com/handsontable/handsontable/issues/8795)
-- Fixed an issue with checkboxes positions. [#8797](https://github.com/handsontable/handsontable/issues/8797)
-- Fixed an issue with opening position of the dropdown (autocomplete) editor. [#8802](https://github.com/handsontable/handsontable/issues/8802)
-- Fixed an issue with time cell rendering for RTL [#8805](https://github.com/handsontable/handsontable/issues/8805)
-- Fixed an issue with context menu in RTL mode [#8825](https://github.com/handsontable/handsontable/issues/8825)
-- Fixed an issue with filters menu UX for RTL [#8828](https://github.com/handsontable/handsontable/issues/8828)
-- Wrong TypeScript definition in the BasePlugin class [#9175](https://github.com/handsontable/handsontable/issues/9175)
-- Fixed an issue with the code examples' deployment workflow, where not all example directories were deployed to gh-pages. [#9182](https://github.com/handsontable/handsontable/issues/9182)
-- *React:* Fixed an issue with registering modules for React wrapper [#9140](https://github.com/handsontable/handsontable/issues/9140)
+### Added
+- Added `ShortcutManager`, a new API for customizing keyboard shortcuts. [#8942](https://github.com/handsontable/handsontable/issues/8942)
+- Added support for RTL languages, by introducing a new configuration option: `layoutDirection`. [#8760](https://github.com/handsontable/handsontable/issues/8760)
+- Added an Arabic translation. [#9208](https://github.com/handsontable/handsontable/issues/9208)
+- Added a new configuration option: `fixedColumnsStart`. [#8760](https://github.com/handsontable/handsontable/issues/8760)
 
 ### Changed
-- **Breaking change**: Changed the way `updateSettings` handles data by introducing `updateData` instead of `loadData` in the majority of cases. [#7263](https://github.com/handsontable/handsontable/issues/7263)
-- **Breaking change**: Changed the way the plugins' `updatePlugin` method reacts to `updateSettings` calls. [#9021](https://github.com/handsontable/handsontable/issues/9021)
-- Changed the Code Examples Deployment workflow to make a dry-run on feature branches (without the actual deployment) [#8980](https://github.com/handsontable/handsontable/issues/8980)
+- **Breaking change**: Changed the way `updateSettings()` handles data, by introducing `updateData()` instead of `loadData()` in most cases. [#7263](https://github.com/handsontable/handsontable/issues/7263)
+- **Breaking change**: Changed the way any plugin's `updatePlugin()` method reacts to `updateSettings()` calls. [#9021](https://github.com/handsontable/handsontable/issues/9021)
```

Comment:
Let's not miss the opportunity to explain what justified making these breaking changes. Right now, it looks as this changes were made for the sake of changing. I propose to change these explanations completely:

- **Breaking change**: Changed the way `updateSettings()` handles data updates. Now, when it is provided with a new data object, it updates the data without resetting any states. It is an improvement of performance and consistency of user experience. [#7263](https://github.com/handsontable/handsontable/issues/7263)
- **Breaking change**: Calls to `updateSettings()` only update plugin state when the options object contains configuration options relevant to the given plugin. It is an improvement of performance and consistency of user experience. [#9021](https://github.com/handsontable/handsontable/issues/9021)

Reply:
@warpech it would be great to see these in the blog post as well

Reply:
@warpech done in [3e9465e](https://github.com/handsontable/handsontable/pull/9271/commits/3e9465e057588ad6b43eafc0905601dae9b474fc)

@krzysztofspilka I updated the blog accordingly